man/search_selection_choices.Rd

```diff
@@ -15,7 +15,7 @@ search_selection_choices(
 )
 }
 \arguments{
-\item{input_id}{Input name. Reactive value is available under input[[input_id]].}
+\item{input_id}{Input name. Reactive value is available under input[\link{input_id}].}
```

Comment:
Why this changed to `\link`

Reply:
All elements inside `[...]` will change to links in the Roxygen Markdown format.  This one should not have happened, and the proper way to write roxygen markdown is to use code blocks, i.e. backticks for code blocks.  I'm not sure if this is going to appear correctly in github comments, but ``input[input_id]]``

Reply:
The backticks disappeared, so substitute quotes for backticks in this workaround `'input[input_id]]'`

Reply:
I think this would do `\code{input[[input_id]]}`, wouldn't it?